src/app/components/Text/index.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import remark from 'remark';
+import reactRenderer from 'remark-react';
 import { arrayOf, shape, string } from 'prop-types';
 
 const Text = ({ blocks }) =>
   blocks.map(({ blockId, model }) => (
-    <p key={blockId}>
-      {model.text}
-    </p>
+    <div key={blockId}>
```

Comment:
There are 2 divs around each paragraph, also visible in: [storybook](http://localhost:9001/?selectedKind=Text&selectedStory=default). I think there should be none